frontend/language/src/nb.json

```diff
@@ -1099,22 +1099,29 @@
   "ux_editor.component_properties.allowPopupsToEscapeSandbox": "Tillat popups for å avslutte sandbox",
   "ux_editor.component_properties.alwaysShowAddButton": "Vis alltid 'Legg til'-knapp",
   "ux_editor.component_properties.attribution": "Opphav",
+  "ux_editor.component_properties.audio": "Lyd",
   "ux_editor.component_properties.autocomplete": "HTML autofullfør",
   "ux_editor.component_properties.autocomplete_default": "Standard",
   "ux_editor.component_properties.bindingToShowInSummary": "Velg hvilket felt som skal vises i oppsummeringen",
   "ux_editor.component_properties.breakAfter": "Sideskift ETTER komponenten",
   "ux_editor.component_properties.breakBefore": "Sideskift FØR komponenten",
   "ux_editor.component_properties.buttonStyle": "Knappens stil",
+  "ux_editor.component_properties.cards": "Kort",
   "ux_editor.component_properties.centerLocation": "Sentrum av kartet",
+  "ux_editor.component_properties.children": "Barn",
+  "ux_editor.component_properties.color": "Farge",
   "ux_editor.component_properties.compact": "Bruk kompakt visning",
   "ux_editor.component_properties.componentRef": "ID på komponenten det gjelder (componentRef)",
   "ux_editor.component_properties.config_is_expression_message": "Denne egenskapen er konfigurert som et uttrykk, og kan foreløpig ikke redigeres her.",
   "ux_editor.component_properties.currency": "Valuta",
   "ux_editor.component_properties.current_task": "Kun gjeldende oppgave",
   "ux_editor.component_properties.dataListId": "Id på dataliste",
+  "ux_editor.component_properties.dataModelBindings": "Datamodellknytninger",
```

Comment:
Is this used? If it comes from the main properties during generating components, I dont think we use this in the autogenerated config. The dataModelBindings is rendered in a custom accordion 🤔

Reply:
I don't think it's used. I just followed the "New component" tutorial slavishly. I'll delete it 😊